### PR TITLE
Adding Hazel to Productivity Section

### PR DESCRIPTION
**Pull request by natelandau**
Hazel is one of the most power automation tools on a mac.  You can find out more here: https://www.noodlesoft.com/hazel.php

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -145,6 +145,8 @@
 - [CloudClip](http://www.thinkbitz.com/cloudclip/) - Sync your clipboard between your Mac and your iOS devices.
 - [Divvy Window Manager](http://mizage.com/divvy/) - Window management for tiling your windows.
 - [f.lux](http://stereopsis.com/flux/) - Automatically adjust your computer screen to match lighting.
+- [Hazel](https://www.noodlesoft.com/hazel.php) - Automated organization
+for your Mac
```
Comment (iCHAIT) [inline README.md]: Why is there a line break?

Also no need to say for your mac.

**Comment by natelandau** [comment]
fixed.  working fast over here.  apologies

**Comment by iCHAIT** [comment]
np, but it still says `for you mac` which is not required.

**Comment by herrbischoff** [comment]
@natelandau: I'd prefer the description "Create rules to automatically keep your files organized." What do you think?

**Comment by natelandau** [comment]
fixed and ready.  Took the suggested text from @herrbischoff .

**Comment by herrbischoff** [comment]
:+1:

**Pull request merged**